Cargo.toml

```diff
@@ -9,9 +9,9 @@ description = "MR. Dog groups and watches merge requests across all your *hubs"
 [dependencies]
 async-trait = "0.1.64"
 clap = { version = "4.1.1", features = ["derive"] }
+colored = "2.0.0"
```

Comment:
is this used? 🤔

Reply:
I meant to use it but forgoth, will remove until we have a use for it, es ques moa

Reply:
Removed